src/main/scala/com/wavesplatform/history/HistoryWriterImpl.scala

```diff
@@ -23,15 +25,15 @@ class HistoryWriterImpl private(file: Option[File], val synchronizationToken: Re
   import HistoryWriterImpl._
 
   private val FeatureApprovalBlocksCount = functionalitySettings.featureCheckBlocksPeriod
-  private val MinBlocksCountToActivateFeature = functionalitySettings.blocksForFeatureActivation
+  private val MinVotesCountToActivateFeature = functionalitySettings.blocksForFeatureActivation
 
   private val db = createMVStore(file)
   private val blockBodyByHeight = Synchronized(db.openMap("blocks", new LogMVMapBuilder[Int, Array[Byte]]))
   private val blockIdByHeight = Synchronized(db.openMap("signatures", new LogMVMapBuilder[Int, ByteStr].valueType(DataTypes.byteStr)))
   private val heightByBlockId = Synchronized(db.openMap("signaturesReverse", new LogMVMapBuilder[ByteStr, Int].keyType(DataTypes.byteStr)))
   private val scoreByHeight = Synchronized(db.openMap("score", new LogMVMapBuilder[Int, BigInt]))
   private val featuresAtHeight = Synchronized(db.openMap("features-at-height", new LogMVMapBuilder[Int, Set[Short]].valueType(DataTypes.featureIds)))
-  private val featuresState = Synchronized(db.openMap("features-state", new LogMVMapBuilder[Int, Map[Short, Byte]].valueType(DataTypes.featureState)))
+  private val featuresState = Synchronized(db.openMap("features-state", new LogMVMapBuilder[Short, Int]))
```

Comment:
How do we differ `approved`/`activated` ?

Reply:
activated = approved + VotingPeriodSize